render/container.go

```diff
@@ -23,7 +23,7 @@ const (
 // NB We only want processes in container _or_ processes with network connections
 // but we need to be careful to ensure we only include each edge once, by only
 // including the ProcessRenderer once.
-var ContainerRenderer = MakeFilter(
+var ContainerRenderer = ApplyDecorators(MakeFilter(
```

Comment:
ContainerRenderer is used at other places.(e.g. `ContainerWithImageNameRenderer`), so I guess this will cause the decorators to be applied multiple times, right? Can we avoid that?

For instance we could only apply the decorators at the toplevel.

In fact, a general strategy for this would be desirable (i.e. make sure that Apply decorators is only applied once at the top level, assuming that's correct and semantically equivalent to what we do)